gdcdictionary/schemas/analyte.yaml

```diff
@@ -168,11 +173,13 @@ properties:
     $ref:
       - "_terms.yaml#/ribosomal_rna_28s_16s_ratio/common"
     type: number
+    minimum: 0
 
   rna_integrity_number:
```

Comment:
ticket has a min (DICT-100)

Reply:
oops - that was for rna_integrity_number

Reply:
Confirmed with Bill the rna_integrity_number minimum value mentioned in ticket needs review. hence did not add it to node. can be added in future if needed after confirming